lib/scenic/themes.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Scenic.Themes do
+  use GenServer
+
+  alias Scenic.Theme
+
+  @ets_name __MODULE__
+
+  @impl GenServer
+  def init(_opts) do
+    table_handle =
+      :ets.new(@ets_name, [:set, :protected, :named_table, {:read_concurrency, true}])
```

Comment:
So, we setup a named table here so that anybody can read from it without the full genserver cost.

I'm curious if this function should also setup a default style pulled from either the settings or some normal Scenic default.

Reply:
I agree that there should be a default scenic theme. That would avoid user code from having to treat the "no theme" case separately.

Reply:
Absolutely yes on the default scene. I vote for `:dark`, but that might be just me...